packages/remix/defaults/server-node.mjs

```diff
@@ -55,7 +56,8 @@ function createRemixRequest(req, res) {
   };
 
   if (req.method !== 'GET' && req.method !== 'HEAD') {
-    init.body = req;
+    init.body = createReadableStreamFromReadable(req);
```

Comment:
This matches the behavior of the `remix-serve` CLI:

https://github.com/remix-run/remix/blob/44d1cc61069ec29cb271f50ea1be58c21979785e/packages/remix-express/server.ts#L111-L112